core/src/main/resources/error/error-classes.json

```diff
@@ -3,6 +3,34 @@
     "message" : [ "Field name %s is ambiguous and has %s matching fields in the struct." ],
     "sqlState" : "42000"
   },
+  "CANNOT_CAST_DATATYPE" : {
+    "message" : [ "Cannot cast %s to %s." ],
+    "sqlState" : "22018"
+  },
+  "CANNOT_CHANGE_DECIMAL_PRECISION" : {
+    "message" : [ "%s cannot be represented as Decimal(%s, %s)." ],
+    "sqlState" : "22023"
+  },
+  "CANNOT_EVALUATE_EXPRESSION" : {
+    "message" : [ "Cannot evaluate expression: %s: %s" ],
+    "sqlState" : "42000"
+  },
+  "CANNOT_GENERATE_CODE_FOR_EXPRESSION" : {
+    "message" : [ "Cannot generate code for expression: %s" ],
+    "sqlState" : "42000"
+  },
+  "CANNOT_PARSE_DECIMAL" : {
+    "message" : [ "Cannot parse any decimal" ],
```

Comment:
This is a strange message. Can we change this to `Cannot parse decimal`?

Reply:
ok